src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -427,6 +430,16 @@ export default {
 			FLSS_ONGOING_EXP_KEY,
 			'EXP-VUE-FLSS-Ongoing-Sitewide'
 		);
+
+		// Recommended row replacement test
+		const { version } = trackExperimentVersion(
```

Comment:
The ticket mentions that this experiment should only run for new users, but this runs for all users. Are we indeed only targeting new users?

Reply:
I think this experiment for logged in and logged out users.
Here the case for logged out users, https://kiva.atlassian.net/browse/CORE-1343
And here the case for logged in users, with personalized copy: https://kiva.atlassian.net/browse/CORE-1342